src/bucket/test/BucketListTests.cpp

```diff
@@ -259,7 +259,6 @@ TEST_CASE("bucket list shadowing pre/post proto 12", "[bucket][bucketlist]")
                 }
             }
         }
-
```

Comment:
Not sure why there is churn here on empty lines. This should be undone.
Actually for test files we should not perform the `assert`->`releaseAssert` substitution as it potentially interferes with test debugging, so you're just better off reverting those changes in `test` and `simulation` folders

Reply:
I will revert all the Test versions to use the normal assert.